### ISSUE TITLE
NTR: NADH-cytochrome b5 reductase low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7941]
New term request NADH-cytochrome b5 reductase low (segal_060817195214):   UMLS: 3550551

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000133
- label: Decreased circulating nicotinamide adenine dinucleotide-cytochrome b5 reductase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:8874222
- def: The activity of nicotinamide adenine dinucleotide (NADH)-cytochrome b5 reductase in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7941
- synonyms: NADH-cytochrome b5 reductase low(NONE)
